bin/pycbc_multi_inspiral

```diff
@@ -757,9 +757,9 @@ with ctx:
                         % len(snr_dict[ifo])
                         for ifo in args.instruments
                     }
-                    # Calculate the coincident and coherent SNR.
-                    # First check there is enough data to compute the SNRs.
-                    if len(coinc_idx) != 0 and nifo > 1:
+                    # Calculate the coincident SNR, regardless of how many IFOs
+                    # are available, only if there are triggers so far.
+                    if len(coinc_idx) != 0:
```

Comment:
It is strange to talk about "coincident SNR" in a one-detector analysis. I think a more correct term is incoherent network SNR, which is valid regardless of the number of detectors. If you agree, please update the comment.

Reply:
Okay, I will update the comment, but I will figure out a way to leave the term "coincident" given the `coinc` in variable names all over the place.